### PR TITLE
add IDEs and Visualizers

### PR DESCRIPTION
**Pull request by noraj**
Signed-off-by: Alexandre ZANNI <16578570+noraj@users.noreply.github.com>

Read [the contribution guidelines](https://github.com/Escape-Technologies/awesome-graphql-security/blob/main/CONTRIBUTING.md)

**Why is this entry awesome?**

More alternatives

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +61,11 @@ A curated list of awesome GraphQL Security frameworks, libraries, software and r
 - [GraphQL Visualizer](https://www.graphqlvisualizer.com) - Visualize GraphQL schema.
 - [Voyager](https://github.com/IvanGoncharov/graphql-voyager) - Represent any GraphQL API as an interactive graph.
 - [GraphQL Inspector](https://github.com/kamilkisiela/graphql-inspector) – Validate schema, get schema change notifications, validate operations, find breaking changes, look for similar types, schema coverage.
+- [GraphQL Rover](https://github.com/Brbb/graphql-rover) - GraphQL schema viewer (requires URL + introspection, can't paste JSON)
+- [Nova](https://github.com/nova-introspection/Nova) / [NovaQL](https://github.com/nova-introspection/novaql) - GraphQL schema viewer (requires URL + introspection, can't paste JSON)
```
Comment (nullswan) [inline README.md]: Theses two entries are not formatted accordingly to awesome-list format. 1 entry per line and no explanation parenthesis.

Also `Nova` is no longer maintained.

File: README.md [hunk 2]
```diff
@@ -60,6 +61,11 @@ A curated list of awesome GraphQL Security frameworks, libraries, software and r
...
+- [GraphQL Schema Viewer](https://github.com/rajinwonderland/graphql-schema-viewer/tree/master/) - GraphQL schema viewer
```
Comment (nullswan) [inline README.md]: this one is likely not maintained as well

File: README.md [hunk 3]
```diff
@@ -60,6 +61,11 @@ A curated list of awesome GraphQL Security frameworks, libraries, software and r
...
+- [CraftQL](https://github.com/yamafaktory/craftql) - CLI GraphQL schema viewer, view schema diagram on the terminal or generate graphviz .dot format file
+- [GraphQL Birdseye](https://github.com/Novvum/graphql-birdseye) - GraphQL schema viewer
```
Comment (nullswan) [inline README.md]: this one is not online anymore, did you tried locally?

File: README.md (hunk 2 above)
Comment (noraj) [reply README.md]: I removed it because it's broken and will never be fixed.

File: README.md (hunk 1 above)
Comment (noraj) [reply README.md]: I fixed the formatting.

File: README.md (hunk 3 above)
Comment (noraj) [reply README.md]: Yes it still works.

**Review by nullswan — APPROVED** [review]

**Pull request merged**